packages/react-aria-components/stories/index.stories.tsx

```diff
@@ -745,7 +811,16 @@ export const TableExample = () => {
   );
 };
 
-export const TableDynamicExample = () => {
+export const TableExampleStory = {
+  render: () => <TableExample />,
+  parameters: {
+    description: {
+      data: 'Note that row 1 is disabled'
```

Comment:
shouldn't there be a data-disabled on row 1?

Reply:
oh interesting, so our SelectionManager has the following: `return this.state.disabledKeys.has(key) && this.state.disabledBehavior === 'all'` and for Table we have the following default `disabledBehavior: props.disabledBehavior || 'selection'`. This means the `isDisabled` that comes from the `useTableRow` hook isn't actually true...

Now I'm wondering if I should make `data-disabled` be applied if the row's key is in the `disabledKeys` or if it should follow the SelectionManager...

Reply:
urgh this is weird, I'm torn between if it would be more confusing to a user if "data-disabled" doesn't follow their "disabledKeys" array or that `data-hovered` wouldn't appear on a hovered row if they had that row in their "disabledKeys" array but they have `disabledBehavior="selection"` (which once again is the default) and thus the row could have a row action and still fire something...

I think I'm leaning towards updating the `useHover` in these components to use `states.isDisabled` perhaps (aka use the SelectionManager's isDisabled)